scripts/cam2telstate.py

```diff
@@ -340,11 +343,13 @@ def parse_streams(self) -> None:
 
     async def get_receptors(self) -> List[str]:
         """Get the list of receptors"""
+        if self._args.receptors is not None:
+            return self._args.receptors.split(',')
```

Comment:
How do you feel about logging the receptors in use at .info level - might be a useful debug.

Reply:
The master controller already logs the config dictionary, which contains the antenna list - do you think it needs to be logged twice?

Reply:
Nope - was just concerned that the passed in args method might differ from the CAM configure request, but that was before reviewing the other code. Happy to leave it...

Reply:
Does that imply that you approve the PR?